system/services/assetManager/ImageMagickService.cfc

```diff
@@ -22,10 +22,19 @@ component displayname="ImageMagick"  {
 		,          string  quality             = "highPerformance"
 		,          boolean maintainAspectRatio = false
 	) {
-		var currentImageInfo = getImageInformation( arguments.asset );
+
+		var imageBinary = arguments.asset;
+		//var checkImageEXIF = $getPresideSetting( "asset-manager", "chk_imageEXIF" );
```

Comment:
Is there a good reason to _not_ check the exif? (i.e. do we need this to be configurable?)